entrypoint.sh

```diff
@@ -24,7 +24,7 @@ eval "$GITHUB_PAGES_CLEANUP_SCRIPT"
 
 echo "⭐️ Copy build from $PROJECT_BUILD_FOLDER"
 cd ../$PROJECT_BUILD_FOLDER
-cp -R * ../$GITHUB_PAGES_REPO_NAME
+cp -R * ../$GITHUB_PAGES_REPO_NAME/$GITHUB_PAGES_RELEASE_FOLDER
```

Comment:
@hpcodecraft Hey 👋  Good shout!

I think we need to add `if` statement here to check where files would copied into `root` folder or into `$GITHUB_PAGES_RELEASE_FOLDER`. With the current change it's always copying into `$GITHUB_PAGES_RELEASE_FOLDER`.

It makes sense to make this variable optional to accommodate for both scenarios. Are happy to make the changes? 😄

Reply:
I was thinking that it would still work when `$GITHUB_PAGES_RELEASE_FOLDER` just is not set, before it would be empty then, resulting in `cp -R * ../$GITHUB_PAGES_REPO_NAME/`.

It's just an assumption though, my shell scripting is not that good really 😅